File: Taller java/src/nivel_3/Invertir_cadena.java
package nivel_3;

import java.util.Scanner;

public class Invertir_Cadena {
    public static String invertirCadena(String cadena) {
        return new StringBuilder(cadena).reverse().toString();
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print(" Ingresa una cadena para invertir: ");
        String entrada = sc.nextLine();
        String resultado = invertirCadena(entrada);
        System.out.println("Cadena invertida: " + resultado);
        sc.close();
    }
}

File: programacion niveles Python/Nivel 3/Invertir_cadena.py
def invertir_cadena(cadena: str) -> str:
    return cadena[::-1]

# main 
if __name__ == "__main__":
    entrada = input("Ingrese una cadena para invertir: ")
    resultado = invertir_cadena(entrada)
    print("Cadena invertida:", resultado)
